README.md

```diff
@@ -1,76 +1 @@
-
-In classical BDD tools for Java like [JBehave](https://jbehave.org) or [Cucumber](https://cukes.info) scenarios are written in plain text files. This allows non-developers to write scenarios, because no programming knowledge is required. To make scenarios executable, developers write so-called step-implementations. To bind plain text to step implementations regular expressions are used. For developers maintaining these executable scenarios has a high overhead that is not required if tests would be directly written in a programming language.
-
-Beside the classical BDD tools there are a number of tools for Java to write BDD tests in a programming language like Groovy ([easyb](http://easyb.org)) or Scala ([ScalaTest](http://www.scalatest.org)). To our knowledge, however, there is no BDD tool where scenarios can be written in plain Java.
-
-Finally, there are specification testing frameworks like [Spock](https://spock.org) (Groovy) or [LambdaBehave](https://richardwarburton.github.io/lambda-behave/) which are very developer-centric and good for unit-testing, but the generated reports are not in Given-When-Then form and are not easily readable by non-developers.
-
-## BDD with JGiven
-
-* Scenarios are written in standard Java code using a fluent, domain-specific API (no extra language like Scala or Groovy needed, no IDE plugin needed)
-* Java method names and parameters are parsed during test execution (no extra annotations needed)
-* Scenarios are executed by either JUnit or TestNG (no extra test runner needed)
-* Scenarios consist of so-called stages, which share state by injection, providing a modular way of writing Scenarios.
-* JGiven generates scenario reports for business owners and domain experts
-
-## Example
-
-```Java
-
-@Test
-public void a_pancake_can_be_fried_out_of_an_egg_milk_and_flour() {
-    given().an_egg().
-        and().some_milk().
-        and().the_ingredient( "flour" );
-
-    when().the_cook_mangles_everything_to_a_dough().
-        and().the_cook_fries_the_dough_in_a_pan();
-
-    then().the_resulting_meal_is_a_pancake();
-}
-```
-
-The above test can be executed like any JUnit test.
-During the execution, JSON files are generated that can then be used afterwards to generate test reports.
-By default, a plain text report is shown in the console, which would look as follows:
-
-```
-Scenario: a pancake can be fried out of an egg milk and flour
-
-  Given an egg
-    And some milk
-    And the ingredient flour
-   When the cook mangles everything to a dough
-    And the cook fries the dough in a pan
-   Then the resulting meal is a pancake
-```
-In addition, you can generate an [HTML Report](https://jgiven.org/jgiven-report/html5/).
-
-## Getting Started
```

Comment:
For the time being I'd like to keep the sections from here on (minus the documents thing of course)